polynote-kernel/src/main/scala/polynote/kernel/Result.scala

```diff
@@ -22,8 +22,28 @@ sealed abstract class ResultCompanion[T <: Result](msgId: Byte) {
   implicit val discriminator: Discriminator[Result, T, Byte] = Discriminator(msgId)
 }
 
-final case class Output(contentType: String, content: String) extends Result
+final case class Output(contentType: String, content: Vector[String]) extends Result
```

Comment:
Just changed Output to keep a collection of lines; they have to get split into lines anyway, and this way we can avoid re-processing lines that have already been processed.